Redazione ottavo sprint (#192)
SevenBitsSwe/7BitsDocs: PB/Documentazione Esterna/Piano_di_Progetto.tex

        \begin{tikzpicture}[overlay]
        
            % Definisce un triangolo blu in basso a destra
            \fill[colorePie] 
                (current page.south east) -- ++(-9cm,0) -- ++(9cm,9cm);
            
            % Inserisce il logo all'interno del triangolo
            \node[anchor=south east, xshift=-0.3cm, yshift=0.3cm] at (current page.south east) {
                \includegraphics[width=4.5cm]{LogoUnipd.png}
            };
        \end{tikzpicture}
        \fi
    }
        

\vfill
\end{titlepage}
\newpage
\clearpage

\setcounter{page}{1}
%registro modifiche
\begin{center}
\textbf{Registro modifiche}\\
\vspace{2mm}
\renewcommand{\arraystretch}{1.5}
\rowcolors{0}{gray!11}{white} % Aggiunge colore alle righe del registro

\begin{longtable}{|>{\centering\arraybackslash}m{1.5cm}|>{\centering\arraybackslash}m{2cm}|>{\centering\arraybackslash}m{2.5cm}|>{\centering\arraybackslash}m{2.5cm}|>{\centering\arraybackslash}m{5cm}|}
\hline
\textbf{Versione} & \textbf{Data} & \textbf{Autore} & \textbf{Verificatore} & \textbf{Descrizione}\\
\endhead
\hline
1.0.2 & 2025-03-17 & Riccardo Piva & Alfredo Rubino & Redazione ottavo sprint$_G$ \ref{ottavo-sprint$_G$}\\
\hline

> 1.0.1 & 2025-03-8 & Giovanni Cristellon & Leonardo Trolese & Redazione settimo sprint$_G$ \ref{settimo-sprint$_G$}\\

Reviewer: ```suggestion
1.0.1 & 2025-03-08 & Giovanni Cristellon & Leonardo Trolese & Redazione settimo sprint$_G$ \ref{settimo-sprint$_G$}\\
```

        rappresentano i costi ancora da sostenere per completare il prodotto e la relativa documentazione$_G$.\\
        Questo è un indicatore positivo per il gruppo, che può confermare la consegna PB del progetto indicativamente per la data 2025-04-04, riprogrammata a seguito della 
        revisione RTB.\\
        Resta tuttavia aperto il problema dell'imprecisione nella stima delle ore di lavoro previste per ogni sprint, che anche in questa iterazione si è rivelata essere
        imprecisa.\\
        
    \newpage
    \subsubsection{Ottavo sprint$_G$}
    \label{ottavo-sprint$_G$}
        
        \paragraph{Durata sprint$_G$}\mbox{}\\
        \vspace{-1.5em}
        \begin{table}[h] 
        \renewcommand{\arraystretch}{1.2}  
        \begin{tabular}{ l l }
            Inizio: & 2025-03-10 \\
            Fine Prevista: & 2025-03-16 \\
            Fine Effettiva: & 2025-03-16 \\
            Giorni di ritardo: & 0 \\
        \end{tabular}
        \end{table}
        \vspace{-2em}
        {\renewcommand{\arraystretch}{1.5}%
        
        \paragraph{Panoramica generale e obiettivi}\mbox{}\vspace{0.4em}

>         Nel corso dell'ottavo sprint il gruppo si propone di proseguire con lo sviluppo del prodotto MVP, cercare di ultimare il diagramma delle classi e continuare con la readzione di documenti.

Reviewer: ```suggestion
        Nel corso dell'ottavo sprint il gruppo si propone di proseguire con lo sviluppo del prodotto MVP, cercare di ultimare il diagramma delle classi e continuare con la redazione dei documenti.
```

        \\
        Il gruppo ha in programma di:
        \begin{itemize}
            \item Procedere con lo sviluppo del prodotto MVP;
            \item Cercare di ultimare il diagramma delle classi;

>             \item Continuare con la stesura della specifica tecnica;

Reviewer: ```suggestion
            \item Continuare con la stesura della Specifica Tecnica;
```

>             \item Continuare con la stesura del manuale utente;

Reviewer: ```suggestion
            \item Continuare con la stesura del Manuale Utente;
```

>             \item Svolgere icontro con professor Cardin per chiarimenti riguardanti la progettazione.

Reviewer: ```suggestion
            \item Svolgere incontro con professor Cardin per chiarimenti riguardanti la progettazione.
```

        \end{itemize}

        Seguono le attività inserite nel backlog per questo periodo:
        \vspace{-0.5em}
        \begin{itemize}
        \setlength\itemsep{-0.2em}

>         \item [-] Continuazione stesura documento specifica tecnica;

Reviewer: ```suggestion
        \item [-] Continuazione stesura documento Specifica Tecnica;
```

>         \item [-] Continuazione stesura documento manuale utente;

Reviewer: ```suggestion
        \item [-] Continuazione stesura documento Manuale Utente;
```

        \item [-] Continuazione sviluppo prodotto MVP;
        \item [-] Svolto incontro con professor Cardin;
        \item [-] Correzione diagrammi delle classi in seguito all'incontro con il professor Cardin.
        \end{itemize}

        \paragraph{Gestione Rischi}\mbox{}
        \vspace{-1em}
        \subparagraph*{Rischi Attesi}\mbox{}
        

>         Relativamente l'ottava iterazione, dato il prosiguimento nello sviluppo del prodotto MVP, con la conseguente produzione di codice, il gruppo ha identificato dei potenziali rischi in cui potrebbe imbattersi:

Reviewer: ```suggestion
        Relativamente all'ottava iterazione, dato il proseguimento nello sviluppo del prodotto MVP e la conseguente produzione di codice, il gruppo ha identificato alcuni potenziali rischi in cui potrebbe imbattersi:
```

            \label{fig:Diagramma circolare della partizione delle ore consuntive per ruolo - Ottavo sprint}
        \end{figure}

        \paragraph{Rendicontazione delle risorse economiche restanti}\mbox{}\vspace{0.4em}
        \begin{figure}[H]
            \centering
            \includegraphics[width=0.6\linewidth]{oreCostiOttavoSprint.png}
            \caption{Consuntivo orario e costi per ruolo - Ottavo sprint}
            \label{fig:Consuntivo orario e costi per ruolo - Ottavo sprint}
        \end{figure}
        
        \paragraph{Retrospettiva}\mbox{}\vspace{0.4em}

        Durante l'ottavo periodo il gruppo ha effettuato una riunione con il professor Cardin per ricevere chiarimenti riguardanti la progettazione del prodotto MVP.
        Ha inoltre continuato con la fase di programmazione e con l'implementazione dei test automatizzati.\\

        \subparagraph{Analisi delle strategie di mitigazione adottate}\mbox{}\vspace{0.4em}

        L'unico problema in cui il gruppo si è imbattuto è stato un errore nella progettazione sorto durante l'incontro con il professor Cardin.

>         Dato l'incremento di ore di lavoro che i vari membri mettono nel progetto, il gruppo è stato in grado di correggere l'errore in tempo e proseguire con lo sviluppo del prodotto$_G$.

Reviewer: ```suggestion
        Grazie all'incremento delle ore produttive per sprint, dovuto anche al decremento della durata degli sprint da due a una settimana, il gruppo è stato in grado di correggere l'errore in tempo e proseguire con lo sviluppo del prodotto$_G$.
```

        \subparagraph{Raffinamento delle stime}\mbox{}\vspace{0.4em}

        In base alle metriche aggiornate all'ottavo periodo il gruppo può dire di essere lievemente in anticipo.
        In base a quanto prodotto fin ora il budget totale stimato per il completamento del progetto è di 12.746,96€, rispetto ai 12.950,00€ inizialmente prestabiliti.

>         Di questi 3.324,40€ rappresentano i costi ancora da sostenere per completare il prodotto e la relativa documentazione.\\

Reviewer: ```suggestion
        Di questa cifra, 3.324,40€ rappresentano i costi ancora da sostenere per completare il prodotto e la relativa documentazione.\\
```

>         Questo è un buon segnale per il gruppo che fa presagire che la consegna PB stimata per il 2025-04-04 sia un obbiettivo raggiungibile.

Reviewer: ```suggestion
        Questo è un buon segnale per il gruppo, facendo presagire che la consegna PB stimata per il 2025-04-04 sia un obbiettivo raggiungibile.
```
